templates/sag-db-sqlserver/README.md

```diff
@@ -81,7 +78,7 @@ docker-compose -f templates/sag-db-sqlserver/docker-compose.yml run --rm test
 The expected values were successfully retrieved after 1 call within 4 seconds.
 ```
 
-You can now use this database for creating instances of webMethods products (IS, MWS, BPM) with the following database connection properties:
+You can now use this database for creating instances of webMethods products (Integration Server, My webMethods Server, Business Process Management) with the following database connection properties:
```

Comment:
You can now use this database for creating instances of webMethods products (Integration Server, My webMethods Server) with the following database connection properties: